src/main/java/uk/gov/hmcts/reform/hmc/model/HearingDetails.java

```diff
@@ -72,8 +72,9 @@ public class HearingDetails {
 
     private Boolean hearingIsLinkedFlag;
 
-    @Size(max = 70, message = ValidationError.AMEND_REASON_CODE_MAX_LENGTH)
-    private String amendReasonCode;
+    @Valid
+    @NotEmpty(message = ValidationError.INVALID_AMEND_REASON_CODE)
+    private List<@Size(max = 70, message = ValidationError.AMEND_REASON_CODE_MAX_LENGTH) String> amendReasonCodes;
```

Comment:
Using standard annotation here rather than having a method in a service class that checks if its null or empty

Reply:
Fixed in https://github.com/hmcts/hmc-cft-hearing-service/pull/242/commits/9060477b00566402258396dd97f0ec1ca4e20604